salt/modules/virt.py

```diff
@@ -1899,20 +1932,17 @@ def update(name,
         need_update = True
 
     # Update the kernel boot parameters
-    boot_tags = ['kernel', 'initrd', 'cmdline']
+    boot_tags = ['kernel', 'initrd', 'cmdline', 'loader', 'nvram']
```

Comment:
You need to be careful here since in the updated XML you may not have the nvram as this is computed